package.json

```diff
@@ -72,10 +72,12 @@
     "@rollup/plugin-babel": "5.3.0",
     "@rollup/plugin-commonjs": "22.0.0",
     "@rollup/plugin-node-resolve": "13.3.0",
+    "@rollup/plugin-replace": "4.0.0",
     "@wojtekmaj/enzyme-adapter-react-17": "0.6.7",
     "babel-loader": "8.2.5",
     "babel-plugin-module-resolver": "4.1.0",
     "babel-plugin-rewire": "1.2.0",
+    "babel-plugin-search-and-replace": "1.1.1",
```

Comment:
This package is required for the tests to work, because Jest is first transpiling 
all the files using Babel and so, the namespace (The main component's class name)should be replaced.